Tests/Common/Scheduling/TimeRulesTests.cs

```diff
@@ -183,11 +183,36 @@ public void ExtendedMarketCloseWithDelta()
             foreach (var time in times)
             {
                 count++;
-                Assert.AreEqual(TimeSpan.FromHours((20 + 5 - .5)%24), time.TimeOfDay);
+                Assert.AreEqual(TimeSpan.FromHours((20 + 5 - .5) % 24), time.TimeOfDay);
             }
             Assert.AreEqual(1, count);
         }
 
+        [TestCase(0)]
+        [TestCase(-1)]
+        public void EveryValidatesTimeSpan(int timeSpanMinutes)
```

Comment:
I tend to prefer phrasing that states what the validation is, something like `EveryThrowsArgumentExceptionWhenIntervalIsLessThanOrEqualToZero` -- it's very wordy, and recently I've been inserting underscores, so something like <method>_<result>_when<case> formatting, so `Every_ThrowsArgumentException_WhenIntervalIsLessthanOrEqualToZero` -- the benefit of this explicitness is two fold, firstly, tests of documentation are a good thing, and secondly, if the tests fail just the name of the failing test usually provides enough information to determine what might have happened (assuming recent changes broke the test). The underscore pattern isn't really used in this code base, so not sure I would recommend switching now unless it's a team decision.